Model/Customer.php

```diff
@@ -174,7 +173,15 @@ public function getLatestCard()
     protected function initializeObject()
     {
         if ($this->getId()) {
-            $this->customerAPI = $this->customerAPI->find($this->getId());
+            $customerAPI = $this->customerAPI->find($this->getId());
```

Comment:
Maybe we could just call `getId()` once and use the result

Reply:
> > 1. Description of change
> > 
> > This PR adds a mechanism that checks if an instance of the customer API object is Error, if yes, then >it refreshes omise customer data. This PR assumes that
> > Note:
> > In case of changing omise account, all saved customer's credit card information becomes >unavailable.
> 
> What does the PR assume? You appear to have ended mid sentence?

@jonrandy so what I meant was:

When you change omise account, register new account with omise, generate private key and public key, then if you change that key in magento shop in payment options, then when customer comes back to the shop he cannot see all his credit cards that were saved before.

Reply:
👍